### ISSUE TITLE
AcmeException when try to delete an account (HTTP 202: Accepted)

### ISSUE THREAD

**Issue by porunov** [shred/acme4j#24]
Hello, 

I noticed that `registration.deactivate()` throws an `AcmeException` :

```
org.shredzone.acme4j.exception.AcmeException: HTTP 202: Accepted
        at org.shredzone.acme4j.connector.DefaultConnection.throwAcmeException(DefaultConnection.java:359)
        at org.shredzone.acme4j.Registration.deactivate(Registration.java:342)

```
Here is logs:
```
14:35:03.544 [main] DEBUG o.s.a.Registration - deactivate
14:35:03.544 [main] DEBUG o.s.a.c.DefaultConnection - POST https://acme-staging.api.letsencrypt.org/acme/reg/528585 with claims: {"resource":"reg","status":"deactivated"}
14:35:03.777 [main] DEBUG o.s.a.c.DefaultConnection - HEADER null: HTTP/1.1 202 Accepted
14:35:03.778 [main] DEBUG o.s.a.c.DefaultConnection - HEADER Server: nginx
14:35:03.778 [main] DEBUG o.s.a.c.DefaultConnection - HEADER Replay-Nonce: 1zzaJPRTDx5pkV_03QPa7nLfINVTX0jyFgKikznRbwQ
14:35:03.778 [main] DEBUG o.s.a.c.DefaultConnection - HEADER Connection: keep-alive
14:35:03.778 [main] DEBUG o.s.a.c.DefaultConnection - HEADER Pragma: no-cache
14:35:03.778 [main] DEBUG o.s.a.c.DefaultConnection - HEADER Boulder-Requester: 528585
14:35:03.778 [main] DEBUG o.s.a.c.DefaultConnection - HEADER Date: Mon, 05 Dec 2016 12:35:03 GMT
14:35:03.778 [main] DEBUG o.s.a.c.DefaultConnection - HEADER Boulder-Request-Id: fa0orevYBC6V-q7aX0uwbRRdkZku5g_FS-djmKKLx-I
14:35:03.778 [main] DEBUG o.s.a.c.DefaultConnection - HEADER Cache-Control: max-age=0, no-cache, no-store
14:35:03.778 [main] DEBUG o.s.a.c.DefaultConnection - HEADER Expires: Mon, 05 Dec 2016 12:35:03 GMT
14:35:03.779 [main] DEBUG o.s.a.c.DefaultConnection - HEADER Content-Length: 643
14:35:03.779 [main] DEBUG o.s.a.c.DefaultConnection - HEADER Link: <https://letsencrypt.org/documents/LE-SA-v1.1.1-August-1-2016.pdf>;rel="terms-of-service"
14:35:03.779 [main] DEBUG o.s.a.c.DefaultConnection - HEADER Link: <https://acme-staging.api.letsencrypt.org/acme/new-authz>;rel="next"
14:35:03.779 [main] DEBUG o.s.a.c.DefaultConnection - HEADER Content-Type: application/json
14:35:03.779 [main] DEBUG o.s.a.c.DefaultConnection - Replay Nonce: 1zzaJPRTDx5pkV_03QPa7nLfINVTX0jyFgKikznRbwQ
14:35:03.783 [main] ERROR c.j.a.c.r.DeactivateAccountCommand - Can not deactivate account
org.shredzone.acme4j.exception.AcmeException: HTTP 202: Accepted
        at org.shredzone.acme4j.connector.DefaultConnection.throwAcmeException(DefaultConnection.java:359)
```

Sincerely,
Alexandr

**Comment by shred** [comment]
Thank you... How nice! According to Let's Encrypt own specifications, the server should return a 200. I will add a fix that also accepts a 202.

**Comment by porunov** [comment]
Thank you too! Will wait for your fix)

**Comment by shred** [comment]
Fixed in a74a0f05319038c8b88828349355b3c97a85a7b9. Will be published with the next relase.
